frontend/components/registrationbox.js

```diff
@@ -18,35 +20,55 @@ export default function Registrationbox() {
   const [secretver, setSecretver] = React.useState("");
   const [error, setError] = React.useState("");
 
-  const submit = async (e) => {
+  const [loginName, setLoginName] = React.useState("");
+  const [loginSecret, setLoginSecret] = React.useState("");
+
+  const checkSignUpPassword = () => {
+    // If password not entered
+    if (secret == "") return "Please enter Password";
+    // If confirm password not entered
+    else if (secretver == "") return "Please enter confirm password";
+    // If Not same return False.
+    else if (secret != secretver) {
+      return "Password did not match.\n Please try again...";
+    }
+  };
+
+  const submitSignUp = async (e) => {
     e.preventDefault();
-    const errorMessage = checkPassword();
+    const errorMessage = checkSignUpPassword();
     if (errorMessage) {
       setError(errorMessage);
     } else {
       setError("Password matches!");
-      const response = await fetch("http://localhost:3000/register", {
-        method: "POST",
-        body: JSON.stringify({
-          userName: username,
-          name,
-          email,
-          password: secret,
-        }),
-        headers: { "Content-Type": "application/json" },
-      });
+      const response = await fetch(
+        `${process.env.NEXT_PUBLIC_BACKEND_URL}/user/register`,
+        {
+          method: "POST",
+          body: JSON.stringify({
+            userName: username,
+            name,
+            email,
+            password: secret,
+          }),
+          headers: { "Content-Type": "application/json" },
+        }
+      );
       const data = await response.json();
     }
   };
 
-  const checkPassword = () => {
-    // If password not entered
-    if (secret == "") return "Please enter Password";
-    // If confirm password not entered
-    else if (secretver == "") return "Please enter confirm password";
-    // If Not same return False.
-    else if (secret != secretver) {
-      return "Password did not match.\n Please try again...";
+  const submitLogin = async (e) => {
+    e.preventDefault();
+    try {
+      await signIn("credentials", {
+        redirect: false,
+        email: loginName,
+        password: loginSecret,
+      });
```

Comment:
The `signIn` function internally triggers the `authenticate` in file `[...nextauth].js`